src/components/shared/CallNodeContextMenu.js

```diff
@@ -552,26 +552,26 @@ export default explicitConnect<{||}, StateProps, DispatchProps>({
     const rightClickedCallNodeInfo = getRightClickedCallNodeInfo(state);
 
     let thread = null;
-    let threadIndex = null;
+    let threadsKey = null;
     let callNodeInfo = null;
     let rightClickedCallNodePath = null;
     let rightClickedCallNodeIndex = null;
 
     if (rightClickedCallNodeInfo !== null) {
-      const selectors = getThreadSelectors(
-        rightClickedCallNodeInfo.threadIndex
+      const selectors = getThreadSelectorsFromThreadsKey(
```

Comment:
Hm, I'm fixing this with my PR actually (welp, looks like broken but it should fix eventually 😄), but right now context menu will always use the uncached version of the selectors for single threads (since we only check merged thread cache in that function.) Do you think we should fix it here, or is it okay to wait until my PR? (It's also true for other callers too)

Reply:
I can do a quick fix here, by calling `getThreadSelectors` at the top of `getThreadSelectorsFromThreadsKey` in the case of one thread.